### PR TITLE
fix: compile aarch64-linux-android binary on linux when enable lto.

### PR DESCRIPTION
**Pull request by fengys1996**
This PR changes `pthread_atfork` from a strong symbol to a weak symbol.

## Related issue
#81

### CODE REVIEW

File: jemalloc-sys/build.rs [hunk 1]
```diff
@@ -338,6 +338,11 @@ fn main() {
         println!("cargo:rustc-link-lib=atomic");
     }
     println!("cargo:rerun-if-changed=jemalloc");
+
+    cc::Build::new()
```
Comment (BusyJay) [inline jemalloc-sys/build.rs:342]: Should only be built for android.

File: jemalloc-sys/src/pthread_atfork.c [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include <stdint.h>
+
+// These symbols are used by jemalloc on android but the really old android
+// we're building on doesn't have them defined, so just make sure the symbols
+// are available.
+__attribute__((weak)) int
+pthread_atfork(uint8_t *prefork __attribute__((unused)),
```
Comment (BusyJay) [inline jemalloc-sys/src/pthread_atfork.c:7]: Should use the definition from https://man7.org/linux/man-pages/man3/pthread_atfork.3.html. The original definition is just for convenient.

**Comment by fengys1996** [comment]
PTAL @BusyJay

**Comment by fengys1996** [comment]
> You need to sign off your commits.

Done.

File: jemalloc-sys/src/pthread_atfork.c [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#include <pthread.h>
```
Comment (BusyJay) [inline jemalloc-sys/src/pthread_atfork.c:1]: I don't think you need to include this header.

**Review by BusyJay — COMMENTED** [review]
Rest LGTM

File: jemalloc-sys/src/pthread_atfork.c (hunk 3 above)
Comment (fengys1996) [reply jemalloc-sys/src/pthread_atfork.c]: Removed.

**Review by BusyJay — APPROVED** [review]

**Comment by BusyJay** [comment]
Thank you!

**Pull request merged**